src/mindmap-view.ts

```diff
@@ -265,10 +265,15 @@ export default class MindmapView extends ItemView {
   async transformMarkdown(markdown: string) {
     let { root, features, frontmatter } = this.transformer.transform(markdown);
 
+    const rootWithTitle =
```

Comment:
- make a new method to that takes root and returns a new root with title, this new method must be named as camelCase.
- Replace the multiple ternary operators with `if` clauses